packages/datatrak-web/src/api/mutations/useLogin.ts

```diff
@@ -35,15 +35,15 @@ export const useLogin = () => {
       },
       onSuccess: async ({ user }) => {
         await queryClient.invalidateQueries();
+        await queryClient.removeQueries();
+
         if (from) {
-          navigate(from, {
-            state: null,
-          });
+          console.log('Navigating to from:', from);
```

Comment:
```suggestion
```

Reply:
Oops, I overlooked it 😅, have removed them now.